### ISSUE TITLE
< and > as next/previous does not work...

### ISSUE THREAD

**Issue by eiranix** [luis-almeida/jPages#31]
I need to use < and > for the next/previous links, but any time I put them in it breaks the script. Are there any solutions for this?

I have tried using "&lt" and "<" and "&#60" with no success...

**Comment by bakicdj** [comment]
I had a same problem. Just copy and paste « » in your script.
http://www.utexas.edu/learn/html/spchar.html

**Comment by eiranix** [comment]
Yeah I actually ended up going with the single version of those: ‹  ›
